iot/http_example/cloudiot_http_example.js

```diff
@@ -17,7 +17,7 @@
 // [START iot_http_includes]
 const fs = require('fs');
 const jwt = require('jsonwebtoken');
-const request = require('requestretry');
+const request = require('retry-request');
```

Comment:
Nit: can we name this something other than `request` (to avoid confusion with the library of the same name)? Maybe `retryRequest?`